tool/lib/utils.dart

```diff
@@ -87,17 +88,26 @@ class CliCommand {
   final bool throwOnException;
 }
 
-Future<void> runProcess(
+Future<String> runProcess(
   ProcessManager processManager,
   CliCommand command, {
   String? workingDirectory,
   String? additionalErrorMessage = '',
 }) async {
+  String stdout = '';
+
   final process = await processManager.spawn(
     command.exe,
     command.args,
     workingDirectory: workingDirectory,
   );
+  unawaited(
+    process.stdout
+        .transform(
+          utf8.decoder,
+        )
+        .forEach((x) => stdout += x),
+  );
```

Comment:
does ProcessManager not have an existing way to get the output?

also, use string interpolation instead `(x) => '$stdout$x'`